@turbaszek Thanks for the advice!
I checked pre-commit and passed the static test.
but I failed in test_celery_command
```
FAILED tests/cli/commands/test_celery_command.py::TestCeleryStopCommand::test_if_right_pid_is_read - SystemExit: 2
FAILED tests/cli/commands/test_celery_command.py::TestCeleryStopCommand::test_same_pid_file_is_used_in_start_and_stop - SystemExit: 2
FAILED tests/cli/commands/test_celery_command.py::TestWorkerStart::test_worker_started_with_required_arguments - SystemExit: 2
```
I will fix it up

--- a/tests/cli/commands/test_celery_command.py
+++ b/tests/cli/commands/test_celery_command.py
@@ -15,7 +15,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-import importlib
 import unittest
 from argparse import Namespace
 from tempfile import NamedTemporaryFile
@@ -69,15 +68,11 @@ class TestWorkerPrecheck(unittest.TestCase):
 class TestWorkerServeLogs(unittest.TestCase):
 
     @classmethod
-    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def setUpClass(cls):
-        importlib.reload(cli_parser)
         cls.parser = cli_parser.get_parser()
 
-    def tearDown(self):
-        importlib.reload(cli_parser)
-
     @mock.patch('airflow.cli.commands.celery_command.worker_bin')
+    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def test_serve_logs_on_worker_start(self, mock_worker):
         with mock.patch('airflow.cli.commands.celery_command.Process') as mock_process:
             args = self.parser.parse_args(['celery', 'worker', '--concurrency', '1'])
@@ -88,6 +83,7 @@ class TestWorkerServeLogs(unittest.TestCase):
                 mock_process.assert_called()
 
     @mock.patch('airflow.cli.commands.celery_command.worker_bin')
+    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def test_skip_serve_logs_on_worker_start(self, mock_worker):
         with mock.patch('airflow.cli.commands.celery_command.Process') as mock_popen:
             args = self.parser.parse_args(['celery', 'worker', '--concurrency', '1', '--skip-serve-logs'])
@@ -100,16 +96,12 @@ class TestWorkerServeLogs(unittest.TestCase):
 
 class TestCeleryStopCommand(unittest.TestCase):
     @classmethod
-    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def setUpClass(cls):
-        importlib.reload(cli_parser)
         cls.parser = cli_parser.get_parser()
-
-    def tearDown(self):
-        importlib.reload(cli_parser)
 
     @mock.patch("airflow.cli.commands.celery_command.setup_locations")
     @mock.patch("airflow.cli.commands.celery_command.psutil.Process")
+    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def test_if_right_pid_is_read(self, mock_process, mock_setup_locations):
         args = self.parser.parse_args(['celery', 'stop'])
         pid = "123"
@@ -130,6 +122,7 @@ class TestCeleryStopCommand(unittest.TestCase):
     @mock.patch("airflow.cli.commands.celery_command.read_pid_from_pidfile")
     @mock.patch("airflow.cli.commands.celery_command.worker_bin.worker")
     @mock.patch("airflow.cli.commands.celery_command.setup_locations")
+    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def test_same_pid_file_is_used_in_start_and_stop(
         self,
         mock_setup_locations,
@@ -156,17 +149,13 @@ class TestCeleryStopCommand(unittest.TestCase):
 
 class TestWorkerStart(unittest.TestCase):
     @classmethod
-    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def setUpClass(cls):
-        importlib.reload(cli_parser)
         cls.parser = cli_parser.get_parser()
-
-    def tearDown(self):
-        importlib.reload(cli_parser)
 
     @mock.patch("airflow.cli.commands.celery_command.setup_locations")
     @mock.patch('airflow.cli.commands.celery_command.Process')
     @mock.patch('airflow.cli.commands.celery_command.worker_bin')
+    @conf_vars({("core", "executor"): "CeleryExecutor"})
     def test_worker_started_with_required_arguments(self, mock_worker, mock_popen, mock_locations):
         pid_file = "pid_file"
         mock_locations.return_value = (pid_file, None, None, None)

